pkgs/os-specific/bsd/netbsd/default.nix

```
@@ -127,6 +127,10 @@ let
     nativeBuildInputs = [ makeMinimal ];
     buildInputs = [ zlib ];
 
+    # for some reason the build system re-runs configure with HOST_CC
+    depsBuildBuild = [ buildPackages.stdenv.cc ] ++ buildInputs;
+    HOST_CC = "${buildPackages.stdenv.cc}/bin/cc";
```

Shouldn't need full path; it doesn't hurt but this matches the above.
```suggestion
    HOST_CC = "${buildPackages.stdenv.cc.targetPrefix}cc";
```